cmds/main.go

```diff
@@ -134,14 +135,31 @@ func resultWriteCSV(list []lytics.TableWriter, name string) {
 }
 
 func resultWriteTable(cliCtx *cli.Context, list []lytics.TableWriter) {
-	table := termtables.CreateTable()
+	tableString := &strings.Builder{}
+	table := tablewriter.NewWriter(tableString)
 	for i, row := range list {
 		if i == 0 {
-			table.AddHeaders(row.Headers()...)
+			table.SetHeader(rowToString(row.Headers()))
 		}
-		table.AddRow(row.Row()...)
+		table.Append(rowToString(row.Row()))
 	}
-	fmt.Println(table.Render())
+	table.SetAutoFormatHeaders(false)
+	table.Render()
+	fmt.Println(tableString.String())
+}
+
+func rowToString(row []interface{}) []string {
+	res := []string{}
+	for _, val := range row {
+		str, ok := val.(string)
+		if ok {
+			res = append(res, str)
+			continue
+		}
+		res = append(res, fmt.Sprintf("%v", val))
```

Comment:
[NIT] If you're using the default print verb (`%v`) you can use [`Sprint()`](https://godoc.org/fmt#Sprint) here.

```suggestion
		res = append(res, fmt.Sprint(val))
```